lib/matplotlib/__init__.py

```diff
@@ -1199,7 +1200,9 @@ def is_interactive():
 def _init_tests():
     # The version of FreeType to install locally for running the
     # tests.  This must match the value in `setupext.py`
-    LOCAL_FREETYPE_VERSION = '2.6.1'
+    win_arm64 = sys.platform.startswith('win') and \
```

Comment:
Can you drop this change?  Even if the default freetype we will try to build on arm64 in 2.11.1, it is still true that the tests will fail (due to changes in the font rendering) so we should still provide this warning.

I suspect that this (local) variable should have been called `VERSION_OF_FREETYPE_THE_TESTS_WORK_WITH` ;)

Reply:
Yes that make sense 👍

Reply:
Pushed a commit to revert that part.